src/ol/interaction/selectinteraction.js

```diff
@@ -140,6 +156,7 @@ ol.interaction.Select.handleEvent = function(mapBrowserEvent) {
   var features = this.featureOverlay_.getFeatures();
   var /** @type {Array.<ol.Feature>} */ deselected = [];
   var /** @type {Array.<ol.Feature>} */ selected = [];
+  var filter = this.filter_;
```

Comment:
I can't remove, filter is executed in another context. I need a reference like with `selected` variable.